safesocket/safesocket_darwin.go

```diff
@@ -46,6 +47,15 @@ func localTCPPortAndTokenMacsys() (port int, token string, err error) {
 	if auth == "" {
 		return 0, "", errors.New("empty auth token in sameuserproof file")
 	}
+
+	// The above files exist forever after the first run. Check that the server
+	// is actually running (though it may still be in a Stopped state).
+	conn, err := net.Dial("tcp", "127.0.0.1:"+portStr)
```

Comment:
Would it make sense to use `DialTimeout` here with a fairly short timeout (<1s)?

Reply:
Switched to DialTimeout in [safesocket: Timeout macsys test dial after 1s, clarify comments.](https://github.com/tailscale/tailscale/pull/9234/commits/c27afd755fcd81345844082427410c62b9f9607d).